library/src/scala/tasty/TypeTest.scala

```diff
@@ -0,0 +1,27 @@
+package scala.tasty
```

Comment:
`TypeTest` isn't specific to TASTy. It should be in `scala.reflect`, just like `ClassTag`.

Reply:
Moved